packages/yourloops/components/chat/chat-widget.tsx

```diff
@@ -179,113 +157,117 @@ function ChatWidget(props: ChatWidgetProps): JSX.Element {
-                    onChange={handleChange}>
-                <Tab className={classes.chatWidgetTab} label={t('chat-footer-reply')} data-testid="chat-card-reply"
-                     onClick={() => { setPrivateMessage(false) }} />
-                <Tab className={classes.chatWidgetTab} label={t('chat-footer-private')} data-testid="chat-card-private"
-                     onClick={() => { setPrivateMessage(true) }} />
-              </Tabs>
-            </div>
-          }
-        </div>
-        <div ref={inputRow} className={classes.chatWidgetInputRow}>
-          <Button id="chat-widget-emoji-button" data-testid="chat-widget-emoji-button" className={classes.iconButton} onClick={() => { setShowPicker(true) }}>
-            <SentimentSatisfiedOutlinedIcon />
-          </Button>
-          <TextField
-            className={classes.chatWidgetInput}
-            id="standard-multiline-flexible"
-            placeholder={t('chat-footer-start-writing')}
-            multiline
-            maxRows={3}
-            value={inputText}
-            onChange={inputHandler}
-            InputLabelProps={{ shrink: false }}
-            data-testid="chat-card-input"
-          />
-          <Button
-            id="chat-widget-send-button"
-            disabled={inputText.length < 1}
-            className={classes.iconButton}
-            arial-label={t('send')}
-            data-testid="chat-card-send"
-            onClick={sendMessage}
+    <GenericDashboardCard
+      avatar={<EmailOutlinedIcon />}
+      title={`${t('chat-messages-header')} ${nbUnread > 0 ? `(+${nbUnread})` : ''}`}
+      data-testid="chat-card"
+    >
+      <Box position="relative">
+        <Box
+          ref={content}
+          className={classes.chatWidgetContent}
+          data-testid="chat-card-messages"
+        >
+          {messages.map((msg): JSX.Element => (
+            <ChatMessage
+              key={msg.id}
+              text={msg.text}
+              privateMsg={msg.private}
+              author={getUserName(msg.user.firstName, msg.user.lastName, msg.user.fullName)}
+              timestamp={msg.timestamp}
+              ack={msg.destAck}
+              isMine={msg.authorId === userId}
+            />
+          ))}
+        </Box>
+        {showPicker &&
+          <div
+            id="chat-widget-emoji-picker"
+            data-testid="chat-widget-emoji-picker"
+            className={classes.chatWidgetEmojiPickerContainer}
+            onKeyDown={onEmojiPickerKeyPress}
           >
-            <SendIcon />
-          </Button>
+            <EmojiPicker
+              width="100%"
+              height="440px"
+              lazyLoadEmojis
+              emojiStyle={EmojiStyle.NATIVE}
+              onEmojiClick={onEmojiClick}
+            />
+          </div>
+        }
+        <div id="chat-widget-footer" className={classes.chatWidgetFooter}>
+            {userRole === UserRoles.hcp &&
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**🤓 nitpick:** ‏You could have used `user.isUserHcp()` here 😄